[2022-05-23 20:14:46,640] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-15T00:00:00+00:00 [queued]>
[2022-05-23 20:14:46,671] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-15T00:00:00+00:00 [queued]>
[2022-05-23 20:14:46,672] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:14:46,672] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:14:46,672] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:14:46,708] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2021-09-15 00:00:00+00:00
[2022-05-23 20:14:46,735] {standard_task_runner.py:52} INFO - Started process 1135 to run task
[2022-05-23 20:14:46,742] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-15T00:00:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbjbcmkci', '--error-file', '/tmp/tmpu94zrf9y']
[2022-05-23 20:14:46,743] {standard_task_runner.py:80} INFO - Job 147: Subtask second_task
[2022-05-23 20:14:46,880] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-15T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:14:47,106] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-15T00:00:00+00:00
[2022-05-23 20:14:47,107] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:14:47,108] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:14:47,130] {subprocess.py:85} INFO - Output:
[2022-05-23 20:14:47,132] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:14:47,136] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:14:47,191] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210915T000000, start_date=20220523T201446, end_date=20220523T201447
[2022-05-23 20:14:47,219] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:14:47,270] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
